source/ProcessManager.Orchestrations/Processes/BRS_021/ForwardMeteredData/V1/Model/ForwardMeteredDataValidInput.cs

```diff
@@ -39,23 +41,23 @@ public record ForwardMeteredDataValidInputV1(
     Instant StartDateTime,
     Instant EndDateTime,
     ActorNumber GridAccessProvider,
-    IReadOnlyCollection<ForwardMeteredDataValidInputV1.MeteredData> MeteredDataList)
+    IReadOnlyCollection<ForwardMeteredDataValidInput.MeteredData> MeteredDataList)
     : IInputParameterDto
 {
-    public static ForwardMeteredDataValidInputV1 From(ForwardMeteredDataInputV1 input)
+    public static ForwardMeteredDataValidInput From(ForwardMeteredDataInputV1 input)
     {
         var meteredDataList = input.MeteredDataList
             .Select(e => new MeteredData(
                 Position: int.Parse(e.Position!),
-                EnergyQuantity: decimal.Parse(e.EnergyQuantity!),
+                EnergyQuantity: decimal.TryParse(e.EnergyQuantity, NumberFormatInfo.InvariantInfo, out var energy) ? energy : null,
```

Comment:
Since we know that parsing is valid, we could probably also just do:
```suggestion
                EnergyQuantity: e.EnergyQuantity != null ? decimal.Parse(e.EnergyQuantity, NumberFormatInfo.InvariantInfo) : null,
```